wire-ios-data-model/Source/Model/Message/ZMClientMessage+Encryption.swift

```diff
@@ -431,6 +431,7 @@ extension GenericMessage {
         } else if client.failedToEstablishSession {
             // If the session is corrupted, we will send a special payload.
             let data = ZMFailedToCreateEncryptedMessagePayloadString.data(using: String.Encoding.utf8)!
+            logger.error("Failed to create encryptedMessage, session is not established with a client \(data)")
```

Comment:
Suggestion: "failed to encrypt payload: session is not established with client: \(client.remoteIdentifier)"